src/blueapi/worker/task_worker.py

```diff
@@ -48,6 +61,7 @@ class TrackableTask(BlueapiBaseModel, Generic[T]):
     """
 
     task_id: str
+    request_id: str = ""
```

Comment:
Nit: Should ideally have fields with default values after fields with no default values. I actually thought pydantic enforced this, `dataclass` does.

Reply:
Done